src/compile.rs

```diff
@@ -1049,13 +1055,6 @@ impl ByteClassSet {
     }
 }
 
-fn u32_to_usize(n: u32) -> usize {
```

Comment:
It's a guard against the case when `usize` is less than 32 bits. I'm not sure what the state of 16 bit usizes is in rustc, but IIRC people were interested in making it happen. It's certainly possible, so it seems worthwhile to be conservative.